packages/nodes/mention/src/moveSelectionByOffset.ts

```diff
@@ -23,4 +25,14 @@ export const moveSelectionByOffset = (
     Transforms.move(editor, { unit: 'offset' });
     return true;
   }
+
+  if (event.key === 'Escape') {
```

Comment:
This should be extracted to it's own handler. The reason is, this isn't "moving selection by offset". This piece of functionality might not be triggered if the query param is different either.